patterns/1-initial/walk-the-innersource-talk.md

```diff
@@ -150,3 +151,10 @@ practices, accelerating InnerSource adoption organically across departments.
 - Ercan Uçan (Siemens AG)
 - Fabio Huser (Siemens AG)
 - Max Wittig (Siemens AG)
+
+## Alias
+
+- Show, don't tell
+- Lead by example
+- Dogfooding the InnerSource behaviors
+- Role-modeling the InnerSource behaviors
```

Comment:
nit: I see both `role-modeling` and `role-modelling`, not sure if there is precedent already for which one to choose consistently :)

Reply:
I would go for `role-modelling` since we have it twice in other commits, WDYT @spier ?

Reply:
Will change to `role-modelling` as suggested